contrib/juliac-buildscript.jl

```diff
@@ -132,15 +135,8 @@ end
     mapreduce_empty(::typeof(abs), op::F, T) where {F}     = abs(reduce_empty(op, T))
     mapreduce_empty(::typeof(abs2), op::F, T) where {F}    = abs2(reduce_empty(op, T))
 end
-@eval Base.Unicode begin
-    function utf8proc_map(str::Union{String,SubString{String}}, options::Integer, chartransform::F = identity) where F
-        nwords = utf8proc_decompose(str, options, C_NULL, 0, chartransform)
-        buffer = Base.StringVector(nwords*4)
-        nwords = utf8proc_decompose(str, options, buffer, nwords, chartransform)
-        nbytes = ccall(:utf8proc_reencode, Int, (Ptr{UInt8}, Int, Cint), buffer, nwords, options)
-        nbytes < 0 && utf8proc_error(nbytes)
-        return String(resize!(buffer, nbytes))
-    end
+@eval Base.Sys begin
+    __init_build() = nothing
```

Comment:
Similarly here - this code is responsible for setting up `Sys.BINDIR` and `Sys.STDLIB`

Not sure we should just be leaving them uninitialized